apps/wallet-mobile/src/features/SetupWallet/common/StepperProgress/Number1.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import Svg, {Path} from 'react-native-svg'
+
+export const Number1 = () => {
```

Comment:
I don't like this idea of the stepper `number` indicator to be an image, let's use it for now, but definitely worth a refactoring. It's fine for done, error, cancel, but for the progress, it looks odd.

Reply:
The problem is that these images do not look good because the detail needs to be adjusted to the pixel. When using bold, extra pixels are introduced that distort the image.

Reply:
and that is the origin of the issue